src/realm_dart.cpp

```diff
@@ -148,3 +148,8 @@ RLM_API void realm_set_auto_refresh(realm_t* realm, bool enable) {
 RLM_API const char* realm_get_library_cpu_arch() {
     return cpuArch.c_str();
 }
+
+RLM_API void realm_dart_log(realm_log_level_e level, const char* message) {
+    auto logger = realm::util::Logger::get_default_logger();
```

Comment:
We should not be doing this. Why not call the `realm_dart_logger_callback` directly with a message.

Reply:
I mean expose `realm_dart_log_message_for_testing` and from that call `realm_dart_logger_callback`. Also these should be in `realm_dart_logger.cpp`

Reply:
`calucale_minimum_log_level` is not called when logging

Reply:
Yes, but it is called when setting the level. If we set a wrong level to core you will never catch it, since you will skip the core logger.